spec/dataframe_spec.rb

```diff
@@ -1820,6 +1820,27 @@
     end
   end
 
+  context 'unique' do
+    subject do
+      Daru::DataFrame.new({
+        a: [1, 2, 3, 4, 3, 5],
+        b: ['a', 'b', 'c', 'd', 'c', 'c']
+      })
+    end
+
+    context 'with no args' do
+      before { subject.unique }
+      its(:'a.to_a') { is_expected.to eq [1, 2, 3, 4, 5] }
+      its(:'b.to_a') { is_expected.to eq ['a', 'b', 'c', 'd', 'c'] }
+    end
+
+    context 'given a vector' do
+      before { subject.unique(:b) }
+      its(:'a.to_a') { is_expected.to eq [1, 2, 3, 4] }
+      its(:'b.to_a') { is_expected.to eq ['a', 'b', 'c', 'd'] }
+    end
```

Comment:
This spec contradicts method definition, which implies array of vector names should be passed.